src/Data/Hashable/Class.hs

```diff
@@ -664,15 +663,14 @@ instance Hashable B.ByteString where
 
 instance Hashable BL.ByteString where
     hashWithSalt salt bstxt =
-      unsafePerformIO $
-      initializeState k0 k1 $ \state ->
+      unsafeDupablePerformIO $
```

Comment:
unsafeDupableIO is wrong here. `BL.ByteString` can be long/big. It's better to pay price of entering the computation only once, then trying to benefit from a slight `unsafeDupableIO` speed up.

Reply:
Got it thanks :+1: